functions/system.bash

```diff
@@ -38,7 +38,7 @@ basic_packages() {
 
   if cond_redirect apt-get install --yes screen vim nano mc vfu bash-completion coreutils \
     htop curl wget multitail git util-linux bzip2 zip unzip xz-utils cpufrequtils lsb-release \
-    software-properties-common man-db whiptail acl usbutils dirmngr arping; \
+    software-properties-common man-db whiptail acl usbutils dirmngr arping apt-utils; \
```

Comment:
why? hasn't that been in ?

Reply:
No, I noticed that we were getining delayed configuration because it wasn't in so I added it because it should have already been in.